storage/innobase/row/row0import.cc

```diff
@@ -4332,16 +4625,28 @@ row_import_for_mysql(
 
 	row_import	cfg;
 	THD* thd = trx->mysql_thd;
-
 	err = row_import_read_cfg(table, thd, cfg);
 
 	/* Check if the table column definitions match the contents
 	of the config file. */
 
 	if (err == DB_SUCCESS) {
 
+		if (cfg.need_hidden_fts(table)) {
+			dict_table_t *fts_table =
+				build_fts_hidden_table(
+					table, thd, cfg);
+			if (fts_table == nullptr) {
+				return row_import_error(
+						prebuilt, DB_ERROR);
+			}
+			table = fts_table;
+			cfg.m_table = fts_table;
```

Comment:
We do not seem to need the local variable `fts_table`:
```c++
cfg.m_table = table = build_fts_hidden_table(table, thd, cfg);
if (!table) {
	err = DB_ERROR;
	goto import_error;
}
```
Do we even need `table`, or could we use `cfg.m_table` instead?